common/org.testeditor.dsl.common/src/org/testeditor/dsl/common/util/MavenExecutor.java

```diff
@@ -127,7 +127,7 @@ private String getPathToMavenExecutable(String mavenHome, boolean isOsWindows) {
 		if (isOsWindows) {
 			return mavenHome + "\\bin\\mvn.bat";
 		} else {
-			return mavenHome + "/bin/mvn";
+			return (mavenHome + "/bin/mvn").replaceAll("//", "/");
```

Comment:
Thanks. This was an intermediate try to figure out a problem that I had when executing the mvn command line. Will be removed!